internal/storage/caching/fast_stat_bucket_test.go

```diff
@@ -783,3 +784,25 @@ func (t *StatObjectTest) TestRenameFolder() {
 	AssertEq(nil, err)
 	ExpectEq(result, folder)
 }
+
+func (t *StatObjectTest) TestShouldDeleteFolder() {
```

Comment:
nit: you can renamed the test like this - Test_DeleteFolder_Success, Test_DeleteFolder_Failure

Reply:
Done